forge-gui/res/cardsfolder/upcoming/mine_worker.txt

```diff
@@ -0,0 +1,13 @@
+Name:Mine Worker
+ManaCost:2
+Types:Artifact Creature Assembly-Worker
+PT:2/1
+A:AB$ Branch | Cost$ T | BranchConditionSVar$ UrzaCheck | BranchConditionSVarCompare$ GE2 | FalseSubAbility$ GainLife | TrueSubAbility$ GainLifeUrza | SpellDescription$ You gain 1 life. If you control creatures named Power Plant Worker and Tower Worker, you gain 3 life instead.
```

Comment:
I think this one could be simplified to a single GainLife line with `GainLife$ Z` and reusing lines 8 and 9 from your Tower Worker